entgql/annotation.go

```diff
@@ -46,13 +46,29 @@ func OrderField(name string) Annotation {
 }
 
 // Bind returns a binding annotation.
-func Bind() Annotation {
-	return Annotation{Bind: true}
+//
+// No-op function to avoid breaking the existing schema.
+// You can safely remove this function from your scheme.
+// To disable Bind, use BindDisabled()
+func Bind(binds ...bool) Annotation {
+	if len(binds) > 0 && !binds[0] {
+		return Annotation{BindDisabled: true}
+	}
+	return Annotation{}
+}
+
+// BindDisabled returns a disabled binding annotation.
+func BindDisabled() Annotation {
+	return Annotation{BindDisabled: true}
 }
```

Comment:
Don't want to be annoying, but what do you think about `Unbind()` instead of `BindDisabled`? I'm very picky about the public API. If we add `Unbind` I think the change in `Bind` is not needed.

Reply:
I'm okay with that name, update late. ;)

Reply:
But, Bind() still No-op function, right?

Reply:
Right.